stl/inc/yvals_core.h

```diff
@@ -1162,6 +1162,7 @@
 #define __cpp_lib_math_constants          201907L
 #define __cpp_lib_remove_cvref            201711L
 #define __cpp_lib_shift                   201806L
+#define __cpp_lib_smart_ptr_for_overwrite 201811L

```

Comment:
This feature should also be mentioned in the list of `_HAS_CXX20` features at the top of `yvals_core.h`.